Desktop_Interface/isodriver.cpp

```diff
@@ -1337,7 +1337,7 @@ double isoDriver::meanVoltageLast(double seconds, unsigned char channel, int TOP
         break;
     }
 
-    short * tempBuffer = currentBuffer->readBuffer(seconds,1024, 0, 0);
+	std::unique_ptr<short[]> tempBuffer = currentBuffer->readBuffer(seconds,1024, 0, 0);
```

Comment:
Ah, yes. My next pr could be just running stuff through a code formatter, if you find that useful. (I find that type of pr's annoying since they pollute blame and the commit history, but having consistent formatting can really help readability)